doc/module-docs/cc_raspberry_pi/data.yaml

```diff
@@ -11,13 +11,16 @@ cc_raspberry_pi:
       This example will enable the SPI and I2C interfaces on Raspberry Pi.
     file: cc_raspberry_pi/example1.yaml
   - comment: >
-      This example will enable the serial interface on Raspberry Pi.
+      This example will enable the serial console (login shell) on Raspberry Pi.
+      On models prior to Pi 5, enabling the console also enables the UART hardware.
     file: cc_raspberry_pi/example2.yaml
   - comment: >
-      This example will enable the serial interface on Raspberry Pi 5 and disable the UART hardware while enabling the console.
+      This example will enable the serial console on Raspberry Pi 5
+      while disabling the UART hardware (only Pi 5 allows this combination).
```

Comment:
Perhaps it's worth mentioning that (IIRC) this is because Pi 5 runs the serial console on a **separate** UART? (the one between the microHDMI ports)